donate/payments/views.py

```diff
@@ -326,8 +325,10 @@ def form_valid(self, form, send_data_to_basket=True):
 
```

Comment:
```suggestion
start_date = now().date() + relativedelta(months=1)

```

Reply:
This would start the subscription one month from today, instead of today? Is that desired behaviour @alanmoo? it's what we do for upsells, but I think that for monthly donations (not upsell) you'd want to start it immediately?

Reply:
That's correct, we want monthly donations to start immediately.